changelog/change_env_local_to_handle_negated_conditions.md

```diff
@@ -0,0 +1 @@
+* [#1373](https://github.com/rubocop/rubocop-rails/pull/1373): Change `Rails/EnvLocal` to handle negated conditions. ([@fatkodima][])
```

Comment:
```suggestion
* [#1374](https://github.com/rubocop/rubocop-rails/pull/1374): Change `Rails/EnvLocal` to handle negated conditions. ([@fatkodima][])
```

Reply:
Updated.
I wonder how that happened that https://github.com/rubocop/rubocop-rails/issues/1373 is from 2018.

Reply:
Since this issue was transferred from rubocop/rubocop, it is not in chronological order.